src/types.js

```diff
@@ -152,14 +168,16 @@ export type OfferType = {
   createdDate: string
 }
 
-export type JsonLanguageType = { code: string, native_name: string }
+export type JsonLanguageType = { code: string, native_name: ?any, name: ?any }
 
 export type JsonCityType = {
   name: string,
   path: string,
   live: boolean,
   events: boolean,
   extras: boolean,
+  tunews: boolean,
+  ["push-notifications"]: boolean,
```

Comment:
Copied these comments from other PR:

Reply:
Phew, that seems pretty inconsistent. We should use an underscore here (just like with name_without_prefix).
@klinzo @maxammann Do you agree ?

Reply:
I agree.
@svenseeberg can we change this?

If yes then I think it should be easy to fix in this PR.

Reply:
Yes, we can.

Reply:
Fixed